### PR TITLE
Add a page for comparing backend outputs

### PR DESCRIPTION
**Pull request by jwiggins**
Mentioned on #651, but there's no issue.

### CODE REVIEW

File: enable/gcbench/bench.py [hunk 1]
```diff
@@ -122,9 +122,9 @@ def gen_timings(gc, func):
 
     times = np.array(times)
     return {
-        "max": times.max() * 1000,
-        "min": times.min() * 1000,
         "mean": times.mean() * 1000,
+        "min": times.min() * 1000,
+        "max": times.max() * 1000,
```
Comment (jwiggins) [inline enable/gcbench/bench.py:127]: I'm using the order of this dictionary when turning it into an HTML table

Comment (rahulporuri) [reply enable/gcbench/bench.py:127]: should be fine now that we only support python >= 3.6.

**Review by rahulporuri — APPROVED** [review]
I didn't thoroughly examine the code but I did check the output - which looks awesome. Thanks for putting this together! LGTM.

**Comment by jwiggins** [comment]
Yeah. I really like seeing all the outputs next to each other. This is really helpful for #648

**Pull request merged**